File: Java/xProjectDemoJava/app/src/main/java/com/example/xprojectdemojava/ui/send/SendViewModel.java
package com.example.xprojectdemojava.ui.send;
//send es Configuracion General
import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SendViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public SendViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Configuracion General");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: xProjectDemo/app/src/main/java/com/example/xprojectdemo/ui/send/SendViewModel.kt
package com.example.xprojectdemo.ui.send

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.example.xprojectdemo.ui.home.mApp

class SendViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is send FragmentZ "+ mApp.xGlobalsVar.image


        mApp.xGlobalsVar.image="popo4"
    }
    val text: LiveData<String> = _text
}